packages/driver/src/cy/commands/sessions/index.ts

```diff
@@ -51,11 +63,7 @@ export default function (Commands, Cypress, cy) {
   })
 
   Commands.addAll({
-    session (id, setup?: Function, options: { validate?: Function } = {}) {
-      if (Cypress.isBrowser('webkit')) {
-        return $errUtils.throwErrByPath('webkit.session')
-      }
-
+    session (id: string | object, setup?: () => void, options: Cypress.SessionOptions = { cacheAcrossSpecs: false }) {
```

Comment:
I think `id` is fine because we stringify the object later:
```
      // stringify deterministically if we were given an object
      id = _.isString(id) ? id : stringifyStable(id)
```